devbox.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packages": [
+    "bash@latest",
+    "clusterctl@1.6.2",
```

Comment:
why not 1.7.2 https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.7.2

Reply:
I actually copied the contents over from CAPX. This can be cleaned up further, IMO. We don't need clusterctl here actually at all.